services/web/server/tests/unit/with_dbs/fast/test_access_to_studies.py

```diff
@@ -241,7 +240,7 @@ def mocks_on_projects_api(mocker) -> Dict:
     """
     state = ProjectState(
         locked=ProjectLocked(
-            value=False, owner=Owner(first_name="Speedy", last_name="Gonzalez")
+            value=False, owner=Owner(user_id="2", first_name="Speedy", last_name="Gonzalez")
```

Comment:
NOTE: here "2" (i.e. string "2") goes throw because pydantic was able to parse and validate, but it should also work with ``2``, ``None`` or not specifying it (with my proposal)

Now, for this test, HOW relevant is for the test (mock) that we have ``user_id=2``?? Did you add that change here because ``Owner`` required ``user_id``? Can we avoid this change if we set it as optional?

Reply:
It is not optional, we need the user_id to be part of the response.